packages/compass-indexes/src/components/create-index-fields/create-index-fields.tsx

```diff
@@ -270,4 +271,4 @@ class CreateIndexFields extends Component<CreateIndexFieldsProps> {
   }
 }
 
-export default CreateIndexFields;
+export default withTheme(CreateIndexFields);
```

Comment:
why is this needed here, aren't all the components used here already bound inside `compass-components`?

Reply:
It's needed currently to pass the `darkMode` to the `ComboBox` wrapper that's in this file. Thanks for bringing it up, I think it would make more sense to move this into `compass-components` along with the extra wrapper as this is the only place that uses that. It is a bit of refactoring to do that though. Would you prefer I do that here?
Previously this `withTheme` was a level higher up and was being passed to this component, so the changes in this pr are at least bringing us closer to there.

Reply:
Going to merge this, we can clean up the combobox usage a bit down the line, we also might be able to avoid it if we end up using the combobox directly from leafygreen instead of our tweaked port.